app/assets/stylesheets/dashboard.scss

```diff
@@ -257,45 +259,59 @@
   }
 }
 
-.charts-container {
-  max-width: 500px;
-  height: 250px;
-  margin: 0 auto;
-}
-
 .analytics-container {
   width: 1400px;
   margin: 0 auto;
 }
 
+.crayons-tabs--analytics {
+  justify-content: center;
+}
+
 .summary-stats {
+  text-align: center;
   display: flex;
-  flex-wrap: wrap;
-  width: 100%;
   justify-content: space-between;
 
-  .card {
-    flex: 1 0 150px;
+  &__col {
+    flex: 1;
+    width: 10%;
   }
 }
 
-.crayons-tabs--analytics {
-  justify-content: flex-end;
+.graphs {
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+
+  &__col {
+    flex: 1;
+    width: 50%;
+  }
 }
 
-.graphs {
-  .row {
-    display: flex;
-    flex-wrap: wrap;
+.centered-graph {
+  margin: 10px auto;
+  width: 100%;
+}
 
-    .card {
-      flex: 1 0;
+@media screen and (max-width: $breakpoint-s) {
+  .summary-stats {
+    display: block;
+    &__col {
+      display: block;
+      width: 90%;
+      margin: 10px auto;
     }
   }
 
-  h2 {
-    margin: 5px;
-    text-align: center;
+  .graphs {
+    display: block;
+    &__col {
+      display: block;
+      width: 90%;
+      margin: 10px auto;
+    }
```

Comment:
These styles look the same to me; can we combine them? We could also extract them into variables (especially `margin: 10px auto`, which shows up a few times :)

Reply:
I think you can just do:

```scss
.summary-stats,
.graphs {
    display: block;
    &__col {
      display: block;
      width: 90%;
      margin: 10px auto;
    }
  }
```

Reply:
Done ✅   thank you both!